src/styles/mixins.scss

```diff
@@ -0,0 +1,5 @@
+@mixin skeleton-mixin {
+  --border-radius: 9999px;
+  --background: var(--ion-color-primary-rgba);
```

Comment:
just an idea. To avoid using --ion-color-primary-rgba we might can use primary color with specific opacity via tailwind. It will be nice to check if it will work.